HOWTO.md

```diff
@@ -88,6 +88,15 @@ Ubuntu 16.04.
    libsbp version number in the C, Python, JavaScript, and LaTeX developer
    documentation are consistent.
 
+   - C: update `c/include/version.h` with new version
+
+   - Python: update `python/setup.py` with new version, change
+     `IS_RELEASED = false` to `IS_RELEASED = true`.
+
+   - JavaScript: update `package.json` and `package-lock.json`.
+
```

Comment:
All of these version files this should happen in makefile automatically after a "make all", should we mention as such?

Reply:
I'm seeing `c/include/version.h`, `package.json`, and `sbp.cabal.m4` update automatically - whereas `package-lock.json` does not

Reply:
~I'm confused... how do these update automatically?~ Never mind... looks like the sbp generator for C updates this.  I suppose it would make sense to update the Python version too?